### PR TITLE
Updating clusters overview enhancement

### PR DESCRIPTION
**Pull request by xenolinux**
This PR addresses the enhancement in the updating clusters overview page as per the suggestions in this PR: #45298

QE ack is not needed.

Version(s): 4.7 only

Issue: [Jira](https://issues.redhat.com/browse/OSDOCS-3023)

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/overview-enhancement/updating/index.html

Additional information:

### CODE REVIEW

File: updating/index.adoc [hunk 1]
```diff
@@ -6,52 +6,52 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4.
+You can update an {product-title} 4 cluster with a single operation by using the web console or the OpenShift CLI (oc).
 
 [id="updating-clusters-overview-understand-update-service"]
 == Understanding the OpenShift Update Service
-xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red{nbsp}Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, see the following information:
+xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, see the following:
```
Comment (nalhadef) [inline updating/index.adoc:13]: Why was the non-breaking space in Red Hat removed?

File: updating/index.adoc [hunk 2]
```diff
@@ -6,52 +6,52 @@ include::_attributes/common-attributes.adoc[]
...
 
 * xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[About the OpenShift Update Service]
 * xref:../updating/understanding-the-update-service.adoc#unmanaged-operators_understanding-the-update-service[Understanding support policy for unmanaged Operators]
 
 [id="updating-clusters-overview-install-and-configure-update-service"]
 == Installing and configuring the OpenShift Update Service
 xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]: Clusters with internet accessibility can access public APIs; clusters in a restricted network are unable to access public APIs for update information. To provide a similar upgrade experience in a restricted network, you can install and configure the OpenShift Update Service locally so that it is available within a disconnected environment.
-For more information, see the following actions:
+For more information, see the following procedures:
```
Comment (nalhadef) [inline updating/index.adoc:21]: Personal preference: I'd write "...see the following:"

File: updating/index.adoc [hunk 3]
```diff
@@ -6,52 +6,52 @@ include::_attributes/common-attributes.adoc[]
...
-xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. For more information, see the following information:
+== Understanding upgrade channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are specific to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install. The `openshift-install` binary file for a specific version of the {product-title} always installs that minor version. For more information, see the following information:
```
Comment (nalhadef) [inline updating/index.adoc:30]: Last sentence: Earlier, something similar was a personal preference. Here, it's more important. We don't want to use "information" twice in a sentence. Use:
"For more information, see the following:"

Comment (xenolinux) [reply updating/index.adoc]: Agree, addressed

File: updating/index.adoc (hunk 2 above)
Comment (xenolinux) [reply updating/index.adoc]: Addressed

File: updating/index.adoc (hunk 1 above)
Comment (xenolinux) [reply updating/index.adoc:13]: May I know the use case of {nbsp}? @nalhadef

Comment (nalhadef) [reply updating/index.adoc:13]: Hi, @xenolinux. This rule must have been documented somewhere or how else would I know of it? I've requested a discussion about adding this to a style guide. In the meantime, you can skip it if you like. Go ahead and ask for a merge. Thanks!

**Pull request merged**